content/zh/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -2005,12 +2005,13 @@ Register the node as schedulable. Won't have any effect if `--register-node` is
 <!--
 Register the node with the given list of taints (comma separated `<key>=<value>:<effect>`). No-op if `--register-node` is `false`. (DEPRECATED: will be removed in a future version)
 -->
-设置本节点的污点标记，格式为 <code>&lt;key&gt;=&lt;value&gt;:&lt;effect&gt;</code>，
-以逗号分隔。当 <code>--register-node</code> 为 false 时此标志无效。
-已弃用：将在未来版本中移除。
+使用给定的污点列表注册节点，格式为逗号分隔的<code> <key>=<value>:<effect></code>。
+如果<code>--register</code> 节点为”false“则无操作。
+(已弃用：将在未来版本中移除。)
```

Comment:
Preview: https://deploy-preview-31477--kubernetes-io-main-staging.netlify.app/zh/docs/reference/command-line-tools-reference/kubelet/

![image](https://user-images.githubusercontent.com/46767780/150728115-27b643d4-db43-49ed-989e-7701c734d9be.png)

---

I cannot review whether Chinese translation is appropriate or not, but can suggest some things:

```suggestion
使用给定的污点列表注册节点，格式为逗号分隔的<code>&lt;key&gt;=&lt;value&gt;:&lt;effect&gt;</code>。
如果<code>--register-node</code> 节点为false则无操作。
已弃用：将在未来版本中移除。
```